engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -308,6 +313,24 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     this.dnmgr = new StreamingContainerManager(dag);
     this.umbilical = new UmbilicalProtocolLocalImpl();
 
+    Method addURL;
+    URLClassLoader loader;
+    try {
+      loader = (URLClassLoader)ClassLoader.getSystemClassLoader();
```

Comment:
ClassLoader do not have addURl method, it's only present in  URLClassLoader.

I get your point that system classloader might OR might not be URLClassLoader.

Also, considering the operators will e deployed in seperate thread in local mode, I change the logic to add URL to Thread context classloader instead of system class loader.